Neos.Neos/Classes/Service/HtmlAugmenter.php

```diff
@@ -77,6 +78,10 @@ protected function getHtmlRootElement($html)
         if ($rootElement === false || $rootElement->length !== 1) {
             return null;
         }
+        // detect whether loadHTML has wrapped plaintext in a p-tag without asking for permission
+        if ($rootElement instanceof \DOMNodeList && $rootElement->item(0)->tagName === 'p' && substr($html, 0, 2) !== '<p') {
```

Comment:
This would break `<P>Foo</P>` (uppercase tags) and plaintexts strings that start with `<p`. Both probably unlikely thus not an issue, but we could use a case insensitive regex instead to make sure